__tests__/app.test.js

```diff
@@ -245,4 +245,113 @@ describe("app", () => {
         });
     });
   });
+
+  describe("PATCH /api/articles/:article_id", () => {
+    test("200: Should accept object in correct form with new vote, and return updated article with an increase in votes", () => {
+      const updatedVotes = { inc_votes: 75 };
+
+      return request(app)
+        .patch("/api/articles/1")
+        .send(updatedVotes)
+        .expect(200)
+        .then((result) => {
+          expect(result.body.article).toBeInstanceOf(Object);
```

Comment:
I would really look into alternative matchers that would help with the testing of objects e.g. objectContaining().

Have a look at the jest documents as chances are the matcher you need will there!